docs/GettingStarted.md

```diff
@@ -2218,7 +2218,7 @@ require 'datadog/statsd'
 require 'ddtrace'
 
 Datadog.configure do |c|
-  # To enable runtime metrics collection, set `true`. Defaults to `false`
+  # To enable runtime metrics collection, set `true`. Defaults to `true`
   # You can also set DD_RUNTIME_METRICS_ENABLED=true to configure this.
   c.runtime_metrics.enabled = true
```

Comment:
I think we should rephrase this section, as the wording made sense for metrics disabled by default, but it doesn't sounds clear anymore with metrics enabled.

Reply:
updated